pinpoint-hbase/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM openjdk:8u342-slim
 
 ARG PINPOINT_VERSION=${PINPOINT_VERSION:-2.5.1}
+ARG HBASE_VERSION=${HBASE_VERSION-1.2.6}
```

Comment:
Hello, you missed the colon on the version
`ARG HBASE_VERSION=${HBASE_VERSION:-1.2.6}`

Reply:
Thank you, I requested PR again. https://github.com/pinpoint-apm/pinpoint-docker/pull/175